tvtk/tests/test_pyface_utils.py

```diff
@@ -0,0 +1,26 @@
+import mock
+import unittest
+
+from pyface.api import OK, FileDialog
+
+
+class TestPopupSave(unittest.TestCase):
+
+    def _make_mock_file_dialog(self):
+        m = mock.Mock(spec=FileDialog)
+        m.open.return_value = OK
+        m.path = 'mock'
+        return m
+
+    def test_popup_save(self):
```

Comment:
you could also write the symmetric test method where 'open' does not return "OK"